dask/array/percentile.py

```diff
@@ -4,6 +4,7 @@
 from numbers import Number
 
 import numpy as np
+from crick import TDigest
```

Comment:
This import should be local to functions using it to prevent crick from being a required dependency.

Reply:
Moved it to be imported only where needed.